src/saving/SaveHelper.cs

```diff
@@ -211,6 +206,26 @@ public static void DeleteSave(string saveName)
         }
     }
 
+    public static void DeleteExcessSaves(string nameStartsWith, int maximumCount)
+    {
+        var currentSaveNames = new List<string>();
+        var allSaveNames = CreateListOfSaves();
+
+        foreach (var save in allSaveNames.AsEnumerable().Reverse())
```

Comment:
Shouldn't the LINQ stuff work as is without AsEnumerable here? Might need to add the LINQ import line at the start of the file.

Reply:
I looked up some tutorials with LINQ. Apparently the Reverse() method requires the caller to be an IEnumerable first, unless I use OrderBy() instead which sorts based on elements in the list (so in this case it would sort by string values).
Here is a snippet of the Reverse() method call:
IEnumerable<string> strings = new List<string> { "first", "then", "and then", "finally" };
// Will contain { "finally", "and then", "then", "first" }
IEnumerable<string> result = strings.Reverse();

Reply:
Seeing as CreateListOfSaves already does sorting, it makes sense to add a last modified first option to it.

Reply:
Oh I see, thanks!